doc/user-guide/ICE_tutorial.md

```diff
@@ -59,9 +59,9 @@ In order to enable signalling we need an instance of [MongooseIM] running with t
 #### Configuration
 
 You can find MongooseIM installation instructions on [this page](Getting-started.md).
-Once you have cloned the repository and compiled the project, you need to modify the `ejabberd.cfg` config file (you can find this file at `$REPO/_build/prod/rel/mongooseim/etc/ejabberd.cfg`, where `$REPO` is a top-level directory of
+Once you have cloned the repository and compiled the project, you need to modify the `mongooseim.cfg` config file (you can find this file at `$REPO/_build/prod/rel/mongooseim/etc/mongooseim.cfg`, where `$REPO` is a top-level directory of
```

Comment:
Once you have cloned the repository and compiled the project, you need to modify the `mongooseim.cfg` config file (you can find this file at `$REPO/_build/prod/rel/mongooseim/etc/mongooseim.cfg`, where `$REPO` is a top-level directory of the cloned repo).